packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -16,7 +17,8 @@ import {
 } from '..'
 
 /* Logger */
-const log = new Logger({ name: 'oe:batch-submitter:init' })
+const destination = createWriteStream({ dsn: process.env.SENTRY_DSN, tracesSampleRate: 1.0 })
```

Comment:
We should minimize usage of `process.env` to a single file (i.e., something like `config.ts`)

Reply:
`run-batch-submitter` is that file for now; the whole package needs some cleanup for i think for now this is the right place

Reply:
`tracesSampleRate` at 1.0 is still quite high

Reply:
^updated to also be 0.05